src/app/states/layouts/page.mdx

```diff
@@ -0,0 +1,67 @@
+export const metadata = {
+  title: 'layouts',
+  description: 'An app state of component layouts. This section explains how to fetch the `layouts` state.',
+}
+
+# layouts
+
+An [app state](/guides/flux-architecture) of the layouts.
+You can obtain the `layouts` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+A map of `[layout_name]: LayoutComponents` where:
```

Comment:
`Record<string, Array<string>>` makes this more readable I think.
(or `Record<string, LayoutComponents>`)

Reply:
Good call out.